src/Confluent.Kafka/Producer.cs

```diff
@@ -124,10 +124,10 @@ private SafeTopicHandle getKafkaTopicHandle(string topic)
 
         private static readonly LibRdKafka.DeliveryReportDelegate DeliveryReportCallback = DeliveryReportCallbackImpl;
 
-        /// <param name="opaque">
+        /// <remarks>
         ///     note: this property is set to that defined in rd_kafka_conf
```

Comment:
no. good point, i do this in more than one place (changed a few). also i forgot to reference the name 'opaque' having changed from param to remarks. the reason for changing to remarks is a warning is generated if you document one param and not the others (even on a private method).